python/hail/docs/getting_started_developing.rst

```diff
@@ -116,7 +116,7 @@ Several Hail tests have additional dependencies:
 
  - `QCTOOL 1.4 <http://www.well.ox.ac.uk/~gav/qctool>`_
 
- - `R 3.3.1 <http://www.r-project.org/>`_ with packages ``jsonlite`` and ``logistf``, which depends on ``mice`` and ``Rcpp``.
+ - `R 3.3.1 <http://www.r-project.org/>`_ with packages ``jsonlite``, ``SKAT`` and ``logistf``, which depends on ``mice`` and ``Rcpp``.
```

Comment:
I think we should remove the transitive dependencies info. R handles installing them as needed, and I bet SKAT has a bunch too.